lib/core/src/server/dev-server.ts

```diff
@@ -213,14 +214,14 @@ const startManager = async ({
 
     if (options.managerCache) {
```

Comment:
Please use `if (options.managerCache && options.cache)` instead of optional chaining here. If will avoid needlessly stringifying the managerConfig.

Reply:
better way. fix it!

Reply:
👍